oak_functions_service/src/lib.rs

```diff
@@ -58,22 +58,31 @@ pub trait Observer {
     fn wasm_invocation(&self, duration: core::time::Duration);
 }
 
+#[cfg(not(test))]
```

Comment:
Could you add a comment explaining what's going on here?

Will this go away soon? It seems quite inconvenient to have to switch the concrete types for testing.

Reply:
Goes away once https://github.com/project-oak/oak/pull/4606 is merged.

Reply:
updated to remove it!